galoisAvx512_amd64.go

```diff
@@ -0,0 +1,181 @@
+//+build !noasm
+//+build !appengine
+//+build !gccgo
+
+// Copyright 2015, Klaus Post, see LICENSE for details.
+// Copyright 2019, Minio, Inc.
+
+package reedsolomon
+
+//go:noescape
+func _galMulAVX512Parallel82(in, out [][]byte, matrix *[(16 + 16) * dimIn * dimOut82]byte, addTo bool)
+
+//go:noescape
+func _galMulAVX512Parallel84(in, out [][]byte, matrix *[(16 + 16) * dimIn * dimOut84]byte, addTo bool)
+
+const dimIn = 8
```

Comment:
Since these are tweakable, it would be great with a small explanation and maybe put them in a `const` block.

Reply:
Have put them in a cost block and added explanation. Note that they should really not be changed since they are tied to the number of input and output rows that the assembly routine process.